rotkehlchen/tests/utils/bitcoin.py

```diff
@@ -0,0 +1,16 @@
+from rotkehlchen.chain.bitcoin.constants import BLOCKSTREAM_BASE_URL
+from rotkehlchen.chain.bitcoin.manager import BitcoinManager
+from rotkehlchen.chain.bitcoin.types import BitcoinTx
+from rotkehlchen.history.events.structures.base import HistoryEvent
+from rotkehlchen.utils.network import request_get
+
+
+def get_decoded_events_of_bitcoin_tx(
+        bitcoin_manager: 'BitcoinManager',
+        tx_id: str,
+) -> list[HistoryEvent]:
+    """Convenience function to query a bitcoin tx by its id and decode it."""
+    bitcoin_manager.refresh_tracked_accounts()
```

Comment:
why not... inside the `BitcoinManager` initialise the `tracked_accounts` from the database.